src/nav/browser.yml

```diff
@@ -1,66 +1,116 @@
 title: Browser monitoring
 path: /docs/browser
 pages:
-  - title: Introduction to browser monitoring
+  - title: Intro to browser monitoring
```

Comment:
Thoughts on "intro" vs. "introduction" vs. "overview"?

Reply:
@a-sassman "Intro" is definitely better for the nav, simply because "Introduction" is so many characters. In that way, too, "Intro" has an advantage over "Overview". I'd probably keep it as "Introduction" in the actual, header title.

I do feel like there's a subtle difference between an introduction and an overview. I don't think I've thought too long about the difference. It's more of an I-know-it-when-I-see-it-thing. :) I guess I think of an overview as an attempt to wholly describe what a thing can do in as brief a space as possible, while an intro doesn't need to have that kind of comprehensiveness.

This doc has qualities of both an intro and an overview, but it feels a bit more like an intro...